Questionnaire/NHSDigital-Questionnaire-Genomics-Example.json

```diff
@@ -1,371 +1,398 @@
+                "type": "display"
+              },
+              {
+                "linkId": "dataStorageResearch",
+                "prefix": "Data storage:",
+                "text": "6. Genomics England will collect different aspects of my health data from the NHS and other data from organisations listed at https://www.genomicsengland.co.uk/privacy-policy/.The collection and analysis of my health data for research will continue across my entire lifetime and beyond.",
+                "type": "display"
+              },
+              {
+                "linkId": "withdrawal",
+                "prefix": "Withdrawal",
+                "text": "7. I can change my mind about taking part at any time.",
+                "type": "display"
+              }
+            ]
+          }
+        ]
+      },
+      {
+        "linkId": "researchInformation",
+        "text": "More information regarding research in the National Genomic Research Library can be found at www.genomicsengland.co.uk.For any further questions, my healthcare professional can provide information.",
+        "type": "display"
+      },
+      {
+        "linkId": "guidetoSignPage3",
+        "text": "Please use page three to indicate your research choices",
+        "type": "display"
+      },
+      {
+        "linkId": "declaration4",
+        "text": "Confirmation of Your Genomic Test and Research Choices",
+        "type": "group",
+        "item": [
+          {
+            "linkId": "confirmation",
+            "text": "I confirm that I have had the opportunity to discuss information about genomic testing, I agree to the genomic test, and my research choice is indicated below.",
+            "type": "group",
+            "item": [
+              {
+                "linkId": "researchConfirmation1",
+                "prefix": "A.",
+                "text": "I have discussed taking part in the National Genomic Research Library. If your answer to A is NO then please ignore B and sign directly below",
+                "type": "boolean",
+                "required": true
+              },
+              {
+                "linkId": "researchConfirmation2",
+                "prefix": "B.",
+                "text": "I agree that my data and remainder sample may contribute to the National Genomic Research Library",
+                "type": "boolean",
+                "enableWhen": [
+                  {
+                    "question": "researchConfirmation1",
+                    "operator": "=",
+                    "answerBoolean": true
+                  }
+                ],
+                "required": true
+              }
+            ]
+          }
+        ]
+      },
+      {
+        "text": "Are you completing this form on behalf of someone?",
+        "linkId": "isRespondentPatientorAttorney",
```

Comment:
I would rename this question to isRespondentAttorney. This would allow the boolean to be unambiguous (e.g. if true, the filler is an Attorney